server/src/models/user.ts

```diff
@@ -220,4 +220,13 @@ export class User {
     { nullable: true },
   )
   notificationConnections: NotificationUserConnection[] | null;
+
+  @Column({ type: Boolean, default: false })
+  jobFound: boolean;
+
+  @Column({ type: String, default: null, nullable: true })
+  jobFoundCompanyName: string | null;
+
+  @Column({ type: String, default: null, nullable: true })
+  jobFoundOfficeLocation: string | null;
```

Comment:
I am not sure this is scalable. what are the use cases. probable we could use existing `employmentHistory` for that.   AFAIU the main purpose to mark current employment and not the fact that student found the job at certain point.

Reply:
> I am not sure this is scalable. what are the use cases. probable we could use existing `employmentHistory` for that. AFAIU the main purpose to mark current employment and not the fact that student found the job at certain point.

I understand this oppositely, but you could be right.
@valerydluski could you please specify this task more precisely or comment on Artem's suggestion?

Reply:
@ThorsAngerVaNeT 
you can make changes without my participation. I am not a business analyst

Reply:
@ThorsAngerVaNeT Lets use `employmentHistory` and save information there.  I don't think we use it now anywhere.
So, please collect title, company, `dateFrom` and mark as `toPresent:true`

Reply:
You may also extend and add `locationName` to there and use the same that we use for searching location during regstration.

Reply:
> @ThorsAngerVaNeT Lets use `employmentHistory` and save information there. I don't think we use it now anywhere. So, please collect title, company, `dateFrom` and mark as `toPresent:true`

Sure, I was going to use `employmentHistory` and redo this task, but I was working on other tasks